samples/testng-parallel-mobile-tests/README.md

```diff
@@ -52,5 +52,5 @@
 
     Run test with your Bitbar api key:
     ```
-    mvn clean test -DBITBAR_API_KEY=<YOUR_BITBAR_API_KEY>
+    mvn clean test -DBITBAR_API_KEY=<YOUR_BITBAR_API_KEY> -Dtestng.dtd.http=true
```

Comment:
What do you think about specifying a DTD using `HTTPS` instead?

Reply:
Yeah I blindly followed solution proposed by logs, instead of simply changing link, sorry.